### DISCUSSION TITLE
Error when I start to train the model

### DISCUSSION THREAD

**Discussion by PauloHirokiNishimoto** [neuralhydrology/neuralhydrology#64] [Q&A]
I already used Neural Hydrology for my tasks, but this time I'm trying to use different dataset and I'm getting this error. Anyone know how to solve it, please?

ValueError: Cannot infer a legal frequency from dataset: None.

My index is in datetime format and all my columns are float64 type.

**Comment by kratzert** [comment]
Hi @PauloHirokiNishimoto 

please use the Issue section for code related problems that might hint to a problem with the code. The discussion section is meant to be used for discussions about applications, future development, best practices etc.

When you open an issue, please provide more information (e.g. a small data sample, your config, the output log, the stack trace) as with this little information it is impossible to find the problem. See the [docs](https://neuralhydrology.readthedocs.io/en/latest/contributing.html#reporting-bugs-and-feature-requests) for more information about error reporting.
